exporter/urls.py

```diff
@@ -3,5 +3,5 @@
 from exporter import views
 
 urlpatterns = [
-    path("api/download_export", views.download_export, name="download-export"),
+    path("publication/<int:id>/download", views.download_export, name="download"),
```

Comment:
Ah yes, I merged them as part of this PR, but then I wasn't sure if they were separated for some reason. I see that there is a middleware that the exporter is using, but I guess we can incorporate it as part of the data_registry app as well.

Reply:
Yes, Datlab had originally kept their logic separate (I suppose because exports are a distinct function), but I don't think it's worth it, since it's so much simpler and safer to look up the active job.